src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -43,7 +43,31 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       chatDiv.scrollTop = chatDiv.scrollHeight;
     };
   </script>
+
+  <script type="text/javascript">
+    function toggle_visibility(id) {
+       var e = document.getElementById(id);
+       if(e.style.display == 'block')
+          e.style.display = 'none';
+       else
+          e.style.display = 'block';
+    }
+</script>
+
 </head>
+<!--
+<div id="popup-box" class="popup-position">
+          <div id="popup-wrapper"> 
+            <div id= "popup-container"> 
+              <h3> Likes </h3>
+              <p> testing this text </p>
+              <p> <%="count"%> </p>
+              <p> line 3 </p>
+              <p> <a href= "javascript:void(0)" onclick="toggle_visibility('popup-box');"> Close</a></p>
+            </div>
+          </div>
+        </div> -->
+
```

Comment:
Could you also remove this, if we want to use it in the future its in a commit so we don't have to worry about losing it :)